src/include/sof/audio/module_adapter/intel/xtensa_overlays.h

```diff
@@ -0,0 +1,15 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ *
+ * Copyright(c) 2022 Intel Corporation. All rights reserved.
+ */
+/*! file xtensa_overlays.h
+ * The source code of xtensa_overlays.h is provided for building modules with
+ * non xtensa toolschains.
+ */
+
+
+void cpu_write_threadptr(int threadptr);
+int cpu_read_threadptr(void);
+
+#define WTHREADPTR(threadptr) cpu_write_threadptr((threadptr))
+#define RTHREADPTR() cpu_read_threadptr()
```

Comment:
This wont work for Zephyr - we need to use Zephyr thread local storage.

Reply:
Yes, this is true. Actually this is not needed to build intel adapter for IADK modules. The headers are taken directly form IADK. Maybe I should extract from them only definitions needed for adapter and drop everything else?

Reply:
xtensa_overlays.h removed from PR. Not needed for intel adapter build.

Reply:
xtensa_overlays.h removed from this PR.